### DISCUSSION TITLE
Establishing a continuous local server connection to make development experience more flawless.

### DISCUSSION THREAD

**Discussion by Aryan-Jadon18** [vercel/next.js#39783] [Ideas]
**What is the problem?**
-I am using VS code as my environment. I have been developing a Next JS app. I switched off my laptop to continue my project the next day. When I opened my VS code and typed npm run dev /yarn dev it shows me the following error:

_**npm WARN config global `--global`, `--local` are deprecated. Use 
`--location=global` instead.
npm ERR! Missing script: "dev"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\acode\AppData\Local\npm-cache\_logs\2022-08-20T16_11_25_212Z-debug-0.log**_

**What is needed to be done?**
-the npm run dev/ yarn dev commands should be able to compile and restart the localhost without any difficulty to the developer.

**Useful links**
[Same issue raised on Freecodecamp](https://forum.freecodecamp.org/t/what-to-do-when-npm-run-dev-is-not-working-react/498167/4)

**Comment by icyJoseph** [comment]
Hi,

This is problem where the location where you run the command. As far as I see from your description, you've run the command in a directory where package.json is missing the `dev` command.

Usually what happens is the VS code terminal opens at the top of the project, and your next application's package.json might not necessarily be there.
